src/web/browser/atomIframe/atomIframe.ts

```diff
@@ -0,0 +1,37 @@
+export const atomIframe = (): Promise<void> => {
+    const iframes: HTMLIFrameElement[] = [].slice.call(
+        document.querySelectorAll('.atom__iframe'),
```

Comment:
I wonder if we should add a separate class for the embeds, just to make sure we keep 'atoms' and 'embeds' distinct? It'd be great for that class to be 'js-' prefixed too, so we know it's *just* for JS behaviour (it should be that for atom__iframe but possible that ship sailed!)

Reply:
I agree with the `js-` prefix but I am scared it might be too late, that this may have knock on effects elsewhere.

I am not sure I see the benefit between the separation between `atoms` and `embeds`. This scrip just cares about resizing iFrames so I think `atom__iframe` is ok (although granted it would be better to highlight the action of resizing/JS execution in the name)

Reply:
It isn't an `atom__iframe` though. We may decide in the future that we change the mechanism of embeds or atoms, and therefore by separating them now, we're making it easier to make that change. I think we should add a new class `js-embed__iframe` and use that

Reply:
Could we do this then as a part of this PR, and add the class @buck06191 needs too as part of this, so we can close https://github.com/guardian/dotcom-rendering/pull/1980 ?

Reply:
done